### PR TITLE
branch2 small changes

### PR DESCRIPTION
**Pull request by AnandAili**

### CODE REVIEW

File: src/main/java/smokefree/Application.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 public class Application {
 
     // Store the application context so classes can use programmatic dependency injection
-    // Hi there - Injecting ApplicationContext programatically
+    // Hi there - Injecting ApplicationContext programatically - Hi there
```
Comment (AnandAiliCS008) [inline src/main/java/smokefree/Application.java]: put proper comment

File: src/main/java/smokefree/AxonFactory.java [hunk 2]
```diff
@@ -126,4 +126,13 @@ public CommandGateway commandGatewaySimple(CommandBus commandBus) {
                 .dispatchInterceptors(newArrayList(new BeanValidationInterceptor<>()))
                 .build();
     }
+
+    @Singleton
+    public CommandGateway commandGatewaySimpleasdasd(CommandBus commandBus) {
```
Comment (AnandAiliCS008) [inline src/main/java/smokefree/AxonFactory.java]: please write proper method names

File: src/main/java/smokefree/Query.java [hunk 3]
```diff
@@ -55,6 +55,13 @@ public long totalVolunteers() {
                 .count();
     }
 
+    public long totalVolunteers12() {
```
Comment (AnandAiliCS008) [inline src/main/java/smokefree/Query.java]: it seems to be duplicated method

**Review by AnandAiliCS008 — CHANGES_REQUESTED** [review]

File: src/main/java/smokefree/Application.java (hunk 1 above)
Comment (AnandAili) [reply src/main/java/smokefree/Application.java]: done

File: src/main/java/smokefree/AxonFactory.java (hunk 2 above)
Comment (AnandAili) [reply src/main/java/smokefree/AxonFactory.java]: done

File: src/main/java/smokefree/Query.java (hunk 3 above)
Comment (AnandAili) [reply src/main/java/smokefree/Query.java]: removed

**Pull request merged**